pkg/server/container_create_windows.go

```diff
@@ -517,19 +513,37 @@ func (c *criService) generateContainerSpec(id string, sandboxID string, sandboxP
 		}
 	}
 
+	// For both LCOW and WCOW, devices are passed through from the host to the
+	// container via the OCI Windows.Devices field
+	addOCIWindowsDevices(&g, config.Devices)
+
 	return g.Config, nil
 }
 
+// getWindowsDeviceInfo is a helper function that returns a spec specified device's
+// prefix and device identifier. A prefix is any string before "://" in the spec
+// device's `HostPath`.
+func getWindowsDeviceInfo(hostPath string) (string, string) {
+	substrings := strings.SplitN(hostPath, "://", 2)
+	if len(substrings) <= 1 {
+		return "", ""
```

Comment:
I think we should return an error in this case

Reply:
Done